shared_model/bindings/CMakeLists.txt

```diff
@@ -40,6 +40,8 @@ if (SWIG_PYTHON OR SWIG_JAVA OR SWIG_CSHARP)
     include_directories(${CMAKE_CURRENT_SOURCE_DIR})
     set_source_files_properties(bindings.i PROPERTIES CPLUSPLUS ON)
     set_property(GLOBAL PROPERTY SWIG_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
+
+    add_dependencies(bindings swig)
```

Comment:
I think it would be better to add dependency for targets like irohajava and not for bindings.

Reply:
Why?

Reply:
Because target bindings does not actually depends from swig.

Reply:
They are for now, nevertheles that branch is taken on:
`if (SWIG_PYTHON OR SWIG_JAVA OR SWIG_CSHARP)`
So I tihnk it's fine